src/cijoe/cli/cli.py

```diff
@@ -637,18 +637,20 @@ def main(args=None):
         if err:
             log.error("Couldn't parse args; exiting")
             return err
+
```

Comment:
This is now also under ``if args is None``,  is that intentional?

Reply:
`args` will always be `None`, when initially running `cijoe` from the command line, so this should always be set. This ensures that the log level is not overwritten by anything that might happen in the call to `create_adhoc_workflow()`, which I would say is the intended behaviour.

Reply:
Right, makes sense.
How about moving the log setup into ``parse_args()``? This ensures that any caller invoking ``parse_args()`` also gets the log level set according to the arguments.

Reply:
Good idea, I've fixed that now